### PR TITLE
Fix F1 help context

### PR DESCRIPTION
**Pull request by vasily-kirichenko**
I broke it in https://github.com/Microsoft/visualfsharp/commit/39dc63fd95e9ee3cabad85dbde05297d1dcab88a

### CODE REVIEW

File: vsintegration/src/FSharp.Editor/Commands/HelpContextService.fs [hunk 1]
```diff
@@ -95,8 +95,8 @@ type internal FSharpHelpContextService
         }
 
     interface IHelpContextService with
-        member this.Language = FSharpCommonConstants.FSharpLanguageName
-        member this.Product = FSharpCommonConstants.FSharpLanguageName
+        member this.Language = "fsharp"
```
Comment (forki) [inline vsintegration/src/FSharp.Editor/Commands/HelpContextService.fs]: urgs. Where is the constant!?

Comment (vasily-kirichenko) [inline vsintegration/src/FSharp.Editor/Commands/HelpContextService.fs]: What would you name it? `FSharpLanguageLongName`?

Comment (forki) [inline vsintegration/src/FSharp.Editor/Commands/HelpContextService.fs]: sounds good

**Comment by vasily-kirichenko** [comment]
@KevinRansom done.

**Comment by KevinRansom** [comment]
Thank you for making this change.

Kevin

**Pull request merged**